src/tools/chip-cert/dacs.py

```diff
@@ -135,15 +135,20 @@ def make_certs_and_keys(self) -> None:
             subject_name = 'Matter Dev PAI 0xFFF1 no PID'
             pid_flag = ''
             type_flag = '-t i'
+            vidpid_fallback_encoding_flag = ''
         elif self.cert_type == CertType.DAC:
             subject_name = 'Matter Dev DAC 0xFFF1/0x{:X}'.format(self.pid)
             pid_flag = '-P 0x{:X}'.format(self.pid)
             type_flag = '-t d'
+            if self.pid >= 0x8010 and self.pid <= 0x8014:
```

Comment:
Add a comment about this test case generation special case.

Reply:
added